src/utils.jl

```diff
@@ -6,7 +6,7 @@ function printpkgstyle(io::IO, cmd::Symbol, text::String, ignore_indent::Bool=fa
     println(io, " ", text)
 end
 
-function linewrap(str::String; io = stdout, padding = 0, width = Base.displaysize(io)[2])
+function linewrap(str::String; io = stdout_f(), padding = 0, width = Base.displaysize(io)[2])
```

Comment:
`io` looks unused here?

Reply:
It's used in the `width` kwarg default. I think I set it up this way so that it was easy to just specify an io, rather than have to figure out the width for yourself, but also allow overriding the width directly. There might be a better way?

Reply:
Ah, indeed. Just leave it for now I guess.